ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/support/ArticleParser.java

```diff
@@ -49,10 +56,12 @@
      * and Category Object will be Recursively placed back into this method.
      *
      * @param evaluatingObject the Json Object to be evaluated.
+     * @param urlParams
      * @return All the articles that were parsed
      */
     @NonNull
-    private static List<Article> jsonObjectHandler(JSONObject evaluatingObject) {
+    private static List<Article> jsonObjectHandler(JSONObject evaluatingObject,
+                                                   Uri urlParams) {
```

Comment:
I would name the parameter `baseUri` instead of `urlParams`. I feel baseUri is more descriptive of what the value actually is.